x-pack/plugins/observability/public/components/slo/burn_rate/alert_details/components/error_rate/error_rate_panel.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiLoadingChart,
+  EuiPanel,
+  EuiStat,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ALERT_RULE_PARAMETERS, ALERT_TIME_RANGE } from '@kbn/rule-data-utils';
+import { GetSLOResponse } from '@kbn/slo-schema';
+import React from 'react';
+import { WindowSchema } from '../../../../../../typings';
+import { useKibana } from '../../../../../../utils/kibana_react';
+import { ErrorRateChart } from '../../../../error_rate_chart';
+import { TimeRange } from '../../../../error_rate_chart/use_lens_definition';
+import { BurnRateAlert } from '../../alert_details_app_section';
+import { getActionGroupFromReason } from '../../utils/alert';
+import { getLastDurationInUnit } from '../../utils/last_duration_i18n';
+
+function getDataTimeRange(
+  timeRange: { gte: string; lte?: string },
+  window: WindowSchema
+): TimeRange {
+  const windowDurationInMs = window.longWindow.value * 60 * 60 * 1000;
+  return {
+    from: new Date(new Date(timeRange.gte).getTime() - windowDurationInMs),
+    to: timeRange.lte ? new Date(timeRange.lte) : new Date(),
+  };
+}
+
+function getAlertTimeRange(timeRange: { gte: string; lte?: string }): TimeRange {
+  return {
+    from: new Date(timeRange.gte),
+    to: timeRange.lte ? new Date(timeRange.lte) : new Date(),
+  };
+}
+
+interface Props {
+  alert: BurnRateAlert;
+  slo?: GetSLOResponse;
+  isLoading: boolean;
+}
+
+export function ErrorRatePanel({ alert, slo, isLoading }: Props) {
+  const {
+    services: { http },
+  } = useKibana();
+
+  const actionGroup = getActionGroupFromReason(alert.reason);
+  const actionGroupWindow = (
+    (alert.fields[ALERT_RULE_PARAMETERS]?.windows ?? []) as WindowSchema[]
+  ).find((window: WindowSchema) => window.actionGroup === actionGroup);
+
+  // @ts-ignore
+  const dataTimeRange = getDataTimeRange(alert.fields[ALERT_TIME_RANGE], actionGroupWindow);
+  // @ts-ignore
+  const alertTimeRange = getAlertTimeRange(alert.fields[ALERT_TIME_RANGE]);
+  const burnRate = alert.fields['kibana.alert.evaluation.value'];
```

Comment:
@kdelemme Is there a const from the alerts as data schema we can use here? I am worried that if a common filed changes, we might have an issue. I was searching into the filed_map [here](https://github.com/elastic/kibana/blob/main/packages/kbn-alerts-as-data-utils/src/field_maps/alert_field_map.ts), but I couldn't find any field related to evaluation. Shouldn't this field defined somewhere? @maryam-saeidi Any idea/input here?

Reply:
I think there is one indeed